server/app.js

```diff
@@ -220,6 +220,10 @@ app.get('/user/:username', async (req, res, next)=>{
 app.get('/edit/:id', asyncHandler(async (req, res, next)=>{
 	res.header('Cache-Control', 'no-cache, no-store');	//reload the latest saved brew when pressing back button, not the cached version before save.
 	const brew = await getBrewFromId(req.params.id, 'edit');
+	if(brew.authors.length > 0 && !(!!req.account && brew.authors.includes(req.account?.username))) {

```

Comment:
If you are just checking for the existence of a variable, I don't think you need to typecast into a boolean; `req.account` should automatically evaluate falsey if it's undefined, etc.

In any case you can simplify with optional chaining even further.

```suggestion
	if(brew.authors.length > 0 && !brew.authors.includes(req?.account?.username)) {

```